### ISSUE TITLE
Accessibility - screen reader issue

### ISSUE THREAD

**Issue by casserole27** [casserole27/jr-developer-project#16]
- screen reader does not read label selection when enter key is pressed

**Comment by casserole27** [comment]
- per Jerry's code review : can produce cleaner code by using the button tag with ARIA attributes
